mlflow_cratedb/adapter/ddl/cratedb.sql

```diff
@@ -59,8 +59,7 @@ CREATE TABLE IF NOT EXISTS "metrics" (
    "timestamp" BIGINT NOT NULL,
    "step" BIGINT NOT NULL,
    "is_nan" BOOLEAN NOT NULL,
-   "run_uuid" TEXT NOT NULL,
-   PRIMARY KEY ("key", "timestamp", "step", "run_uuid", "value", "is_nan")
+   "run_uuid" TEXT NOT NULL
```

Comment:
The test suite started raising `DuplicateKey` exceptions constantly in concurrent logging situations like exercised on behalf of `test_log_metric_concurrent_logging_succeeds`, so the primary key constraint on the "logging" table has been relaxed.